src/main/scala/mesosphere/marathon/tasks/ResourceUtil.scala

```diff
@@ -151,15 +151,26 @@ object ResourceUtil {
     }
   }
 
-  def displayResource(resource: Resource): String = resource.getType match {
-    case Value.Type.SCALAR => s"${resource.getName} ${resource.getScalar.getValue}"
-    case Value.Type.RANGES =>
-      s"${resource.getName} ${
-        resource.getRanges.getRangeList.asScala.map {
-          range => s"${range.getBegin}->${range.getEnd}"
-        }.mkString(",")
-      }"
-    case other: Value.Type => resource.toString
+  def displayResource(resource: Resource): String = {
+    // TODO AW: place it somewhere appropriate
```

Comment:
@aquamatthias What I was actually thinking about is if the user should be able to change that constant. E.g. when he is really interested in all containing ports. Nevertheless I'll put it in as an argument as you suggested for now.